panoptes_aggregation/shape_tools.py

```diff
@@ -11,3 +11,8 @@
     'triangle': ['x', 'y', 'r', 'angle'],
     'fan': ['x', 'y', 'radius', 'spread', 'rotation']
 }
+
+FEEDBACK_STRATEGIES = {
+    'graph2drange': ['x', 'width'],
+    'singleAnswerQuestion': ['answer']
+}
```

Comment:
These have nothing to do with the shape tools, it should belong in its own configuration file with a descriptive name such as `feedback_strategies.py`.